cli/azd/pkg/project/service_target.go

```diff
@@ -135,7 +138,11 @@ func resourceTypeMismatchError(
 // As an example, ContainerAppTarget is able to provision the container app as part of deployment,
 // and thus returns true.
 func (st ServiceTargetKind) SupportsDelayedProvisioning() bool {
-	return st == AksTarget
+	supportedTargets := []ServiceTargetKind{
```

Comment:
Is this new ML target going to be a delayed provisioning thing or was this just a change to make adding additional ones in the future later?